src/EspExceptionDecoder.java

```diff
@@ -96,9 +97,34 @@ public String getMenuTitle() {
     return "ESP Exception Decoder";
   }
 
+  // Need custom one to redirect STDERR to STDOUT
```

Comment:
It would be nice to add something like "Parts of the code are derived from Arduino project" (the source file](https://github.com/arduino/Arduino/blob/2bfe164b9a5835e8cb6e194b928538a9093be333/arduino-core/src/processing/app/helpers/ProcessUtils.java) doesn't have an explicit copyright which we could keep).

Reply:
Added reference to where it came from in latest push.